lib/plugins/aws/package/compile/events/apiGateway/lib/method/integration.js

```diff
@@ -209,7 +209,13 @@ module.exports = {
 
     // set custom request templates if provided
     if (http.request && typeof http.request.template === 'object') {
-      Object.assign(integrationRequestTemplates, http.request.template);
+      Object.keys(http.request.template).forEach(contentType => {
```

Comment:
It might be cleaner to use `_.entries`, as then we have direct eccess to both _key_ and _value_